pyfolio/plotting.py

```diff
@@ -1660,3 +1662,126 @@ def plot_prob_profit_trade(round_trips, ax=None):
+        be generated for each value.
+    color : str, optional
+        Any matplotlib color
+
+    Returns
+    -------
+    ax : matplotlib.Axes
+        The axes that were plotted on.
+    """
+    for std in cone_std:
+        ax.fill_between(aggregate_returns.index,
+                        bounds[float(std)].iloc[:len(aggregate_returns)],
+                        bounds[float(-std)].iloc[:len(aggregate_returns)],
+                        color=color, alpha=0.5)
+    return ax
+
+
+def plot_multistrike_cones(is_returns, oos_returns, num_samples=1000,
+                           name=None, ax=None, cone_std=(1., 1.5, 2.),
+                           random_seed=None):
+    """
+    Plots the upper and lower bounds of an n standard deviation
+    cone of forecasted cumulative returns. This cone is non-parametric,
+    meaning it does not assume that returns are normally distributed. Redraws
+    a new cone when returns fall outside of last cone drawn
+
+    Parameters
+    ----------
+    is_returns : pandas.core.frame.DataFrame
+        Non-cumulative in-sample returns.
+    oos_returns : pandas.core.frame.DataFrame
+        Non-cumulative out-of-sample returns.
+    num_samples : int
+        Number of samples to draw from the in-sample daily returns.
+        Each sample will be an array with length num_days.
+        A higher number of samples will generate a more accurate
+        bootstrap cone.
+    name : str, optional
+        Plot title
+    ax : matplotlib.Axes, optional
+        Axes upon which to plot.
+    cone_std : int, float, or list of int/float
+        Number of standard devations to use in the boundaries of
+        the cone. If multiple values are passed, cone bounds will
+        be generated for each value.
+    random_seed : int
+        Seed for the pseudorandom number generator used by the pandas
+        sample method.
+
+
+    Returns
+    -------
+    ax : matplotlib.Axes
+        The axes that were plotted on.
+    fig : matplotlib.figure
+        The figure instance which contains all the plot elements.
+    """
+    if ax is None:
+        fig = figure.Figure(figsize=(10, 8))
+        FigureCanvasAgg(fig)
+        axes = fig.add_subplot(111)
+    else:
+        axes = ax
+
+    aggregate_returns = timeseries.cum_returns(oos_returns, starting_value=1.)
+    bounds = timeseries.forecast_cone_bootstrap(is_returns,
+                                                len(oos_returns),
+                                                cone_std=cone_std,
+                                                num_samples=num_samples,
+                                                random_seed=random_seed)
+    bounds.index = oos_returns.index
+    plot_cone(aggregate_returns, bounds, axes, cone_std=cone_std)
+
+    bounds_cur = bounds.copy()
+    cone_start = aggregate_returns.index[0]
+
+    aggregate_returns.plot(ax=axes,
+                           lw=3.,
+                           color='k',
```

Comment:
This is a little(very, tbh!) nitpicky, but can we say `'black` instead of `k` here? It's a little more obvious what's going on with the explicit name instead of the shorthand.